openreview/venue_request/venue_request.py

```diff
@@ -828,15 +828,15 @@ def setup_request_form(self):
                     'Program Chairs',
                     'All Senior Area Chairs',
                     'Assigned Senior Area Chair',
-                    'All Area Chairs',
-                    'Assigned Area Chair',
-                    'All Reviewers',
-                    'Assigned Reviewers'
+                    # 'All Area Chairs',
+                    # 'Assigned Area Chair',
+                    # 'All Reviewers',
+                    # 'Assigned Reviewers'
                 ],
                 'default': ['Program Chairs'],
```

Comment:
you need to add the Assigned Senior Area Chair here too

Reply:
I think Celeste's issue wants to have this field not hidden to the user can select the identity options but I do think the default value must contain the assigned SAC, if the users forgets to select it then the SAC can not see their own assignments.

Reply:
Done.